packages/react-test-renderer/index.js

```diff
@@ -9,7 +9,7 @@
 
 'use strict';
 
-var ReactTestRenderer = require('./src/ReactTestRenderer');
+const ReactTestRenderer = require('./src/ReactTestRenderer');
```

Comment:
@gaearon question: files with explict strict mode, should keep `var` right?

Reply:
No, this one is fine.